README.md

```diff
@@ -74,7 +74,7 @@ To be able to use new Capybara theme in your Vue Storefront installation, you ne
    ```
 1. Generate `local.json` file from script `generate-local-config.js`:
    ```
-   node src/themes/vsf-capybara/scripts/generate-local-config.js
+   node src/themes/capybara/scripts/generate-local-config.js
    ```
 1. Update Vue Storefront configuration by copying `local.json` file from `vsf-capybara` to root `config` directory.
```

Comment:
You should also update this part

Reply:
Do you mean "Update Vue Storefront configuration by copying `local.json` file from `vsf-capybara`" point ?

Reply:
Exactly, thanks!